python_modules/dagster/dagster/_core/events/__init__.py

```diff
@@ -1488,6 +1495,62 @@ def __new__(cls, expectation_result: ExpectationResult):
         )
 
 
+@whitelist_for_serdes
+class AssetCheckEvaluationPlannedData(
+    NamedTuple(
+        "_AssetCheckEvaluationPlannedData",
+        [
+            ("asset_key", AssetKey),
+            ("check_name", str),
+        ],
+    )
+):
+    def __new__(cls, asset_key: AssetKey, check_name: str):
+        return super(AssetCheckEvaluationPlannedData, cls).__new__(
+            cls,
+            asset_key=check.inst_param(asset_key, "asset_key", AssetKey),
+            check_name=check.str_param(check_name, "check_name"),
+        )
+
+
+@whitelist_for_serdes
+class AssetCheckEvaluationData(
+    NamedTuple(
+        "_AssetCheckEvaluationData",
+        [
+            ("result", AssetCheckResult),
```

Comment:
This should be an `AssetCheckEvaluation`, not `AssetCheckResult`, right?

Reply:
Renamed this class from AssetCheckEvaluationData to AssetCheckEvaluation, and made it duplicate the fields rather than hold a result object